specs/Standards/Tokens/NonFungibleToken/Event.md

```diff
@@ -134,12 +143,25 @@ EVENT_JSON:{
 }
 ```
 
+Contract metadata update:
+
+```js
+EVENT_JSON:{
+  "standard": "nep171",
+  "version": "1.1.0",
```

Comment:
I wonder if we should tick the version for all the events to 1.1.0, so indexers can make decisions even before they observe the `contract_metadata_update` event, or should we keep it 1.0.0 for all the events given that they have not changed and the new event type is not introducing a breaking change.

Reply:
For every schema change I recommend bumping up the version, keeping the major version the same to indicate there was no breaking change – all following the SemVer.org set of rules.